.github/workflows/github-actions.yml

```
@@ -20,7 +20,7 @@ jobs:
       # If you want to matrix build , you can append the following list.
       matrix:
         go_version:
-          - 1.17
+          - 1.22.1
```

No need to privode patch version.

```suggestion
          - 1.22
```

---

Plus, we could add latest three versions, `1.20`, `1.21` and `1.22` at this moment, on account of compatibility.